draft-ietf-taps-interface.md

```diff
@@ -1273,13 +1441,24 @@ Preconnection.DeframeWith(Deframer)
 {messageData} := Deframer.Deframe(OctetStream, ...)
 ~~~
 
-# Managing Connections: Setting Properties and Obtaining Information {#introspection}
+# Configuring and Querying Connection Properties {#introspection}
```

Comment:
Wait, this got renamed again? 
We changed it because this section also talks about Errors, which are not properties.
Maybe the title isn't ideal, but I suggest we keep the title starting with "Managing Connections".

Reply:
```suggestion
# Managing Connections {#introspection}
```

Reply:
Properties and errors are discussed in the subheads, so maybe just Managing Connections?